bedrock/firefox/views.py

```diff
@@ -443,7 +443,10 @@ def hello(request):
         'zh-TW': 'https://videos.cdn.mozilla.net/uploads/FirefoxHello/firefoxhello_intro_chinese'
     }
 
-    return l10n_utils.render(request, 'firefox/hello/index.html',
+    if (waffle.switch_is_active('firefox-hello-2016')):
```

Comment:
Ah! Not really necessary, yeah. If a switch is not in the database that is the same as `OFF`. So when we're ready to turn it on we'll create it as `ON` and it'll be good. We technically can create one and have it be `OFF`, but there's not much point.